.ci/static-checks.sh

```diff
@@ -175,6 +175,52 @@ check_versions()
 	fi
 }
 
+# Ensure all files (where possible) contain an SPDX license header
+check_license_headers()
+{
+	# See: https://spdx.org/licenses/Apache-2.0.html
+	local -r spdx_tag="SPDX-License-Identifier"
+	local -r spdx_license="Apache-2.0"
+	local -r pattern="${spdx_tag}: ${spdx_license}"
+
+	echo "INFO: Checking for SPDX license headers"
+
+	# List of files to check for a license header inside
+	files=$(git diff-tree \
+		--name-only \
+		--no-commit-id \
+		--diff-filter="ACMRUX" \
```

Comment:
ACMRUX ?

Reply:
new to me - went to look it up - still digesting... https://git-scm.com/docs/git-diff-tree#git-diff-tree---diff-filterACDMRTUXB82308203

Reply:
Yeah, it's a bit opaque :) I've updated the file to make it a bit clearer ;)